exercises/forth/forth_test.py

```diff
@@ -4,13 +4,25 @@
 
 
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.2.0
+# Tests for case-insensitivity are track-specific
+
+class ForthParsingTest(unittest.TestCase):
+    def test_empty_input_empty_stack(self):
+        input_data = [""]
```

Comment:
According to [problem spec](https://github.com/exercism/problem-specifications/blob/84ce49e4002468aa374c5a62a24116650fa9ce6c/exercises/forth/canonical-data.json), this should be
```Python
input_data = []
```

Reply:
Thanks for pointing this out, I've added the correct test now, but I left this particular test because I think that solutions should be able to handle empty lines.

Reply:
I agree. Perhaps this merits a PR to [problem spec](https://github.com/exercism/problem-specifications/blob/84ce49e4002468aa374c5a62a24116650fa9ce6c/exercises/forth/canonical-data.json)?

Reply:
Just about to make one :smile: